### DISCUSSION TITLE
console.log("Hello, world"); in npm package

### DISCUSSION THREAD

**Discussion by ChrisChan888** [AaronDavidNewman/Smoosic#83] [Q&A]
Thank you for creating this amazing tool!!!!
May I ask this support typescript or not?

**Comment by AaronDavidNewman** [comment]
Yes, it is all written in typescript.

**Comment by summerisgood1** [comment]
I would like to help to pack it to npm. However, I have less experience with it. Any idea what part should I start first?

**Comment by AaronDavidNewman** [comment]
Hi Oscar.  There should already be an npm package for it (npm install smoosic/npm run build). 

If you want to contribute to the project, there are many ways.  Look at the readme on the code page and scroll to the end for a list of suggestions.
